mxcubeweb/routes/beamline.py

```diff
@@ -26,7 +26,7 @@ def get_func(name):
             **{"return": getattr(app.mxcubecore.get_adapter(name), attr)(**args)}
         )
 
-        return make_response(result.json(), 200)
+        return make_response(result.dict(), 200)
```

Comment:
How this change leads to the `Content-Type` header being said to `application/json` is just hilarious :joy_cat:

Reply:
Indeed, one would have expected `.json()` to also set `Content-Type` to `application/json`

Reply:
Well, actually, on second thought it somehow makes sense. I guess `json()` returns a string so the content is interpreted as text, while a dictionary always can be interpreted as json when returned as a response

Reply:
This is Flask logic. By default, if you give it a string, it creates a text response. If you give it a dictionary, it makes it a `application/json` response.

Reply:
Yeah, sure it does make sense.